### PR TITLE
Update dependency date-fns to v2.0.0-alpha.27

### LINKED ISSUE #285: Update dependency date-fns to v2.0.0-alpha.27
We use Renovate to update the dependencies of this project. However, we had  to close https://github.com/mozilla-frontend-infra/codetribute/pull/273 since it had breaking changes. We would need a manual update of this package. The [release notes](https://gist.github.com/kossnocorp/a307a464760b405bb78ef5020a4ab136) will be useful for updating the package.

### PR DESCRIPTION
**Pull request by patrickkang**
Fixes #285. 
Reference: https://gist.github.com/kossnocorp/a307a464760b405bb78ef5020a4ab136#v200-alpha27

### CODE REVIEW

**Comment by djmitche** [comment]
I closed and reopened the PR in hopes of getting Netlify to run again -- the logs for its failure don't have much information in them.  If they fail again, or don't run, can you try to figure out why?

**Comment by djmitche** [comment]
Also, Thanks!!

File: src/components/TasksTable/index.jsx [hunk 1]
```diff
@@ -426,7 +426,7 @@ export default class TasksTable extends Component {
                   {item.assignee}
                 </TableCell>
                 <TableCell className={classes.tableCell}>
-                  {formatDistance(item.lastUpdated, new Date(), {
+                  {formatDistance(parseISO(item.lastUpdated), new Date(), {
```
Comment (helfi92) [inline src/components/TasksTable/index.jsx]: 👌

**Comment by patrickkang** [comment]
Thanks for the review @helfi92 and @djmitche. Made changes for sorting dates.

I am not so sure why the first netlify build failed and yet the second one was successful. And now it's back again. Any clue?

**Comment by helfi92** [comment]
@patrickkang If you enjoyed working on this issue, you could help out with migrating date-fns in our other repository https://github.com/taskcluster/taskcluster/issues/705. Feel free to comment on the issue directly if you're interested. Otherwise, it's all good. Thank you for your work!

**Pull request merged**